File: delete_pinecone.py
from pinecone import Pinecone

pc = Pinecone(api_key="pcsk_28r4Ge_BmSFxpNJk26WKRNU5M4UvXp6AP8MgJaSFtuBqdY8sfZTaXZHa39S1bkz9xGbg6X")

index = pc.Index("n8n1536")  # your index name

index.delete(delete_all=True)

print("Deleted successfully")

File: delete_pinecone.js
import { Pinecone } from '@pinecone-database/pinecone';

const pc = new Pinecone({
  apiKey: "pcsk_28r4Ge_BmSFxpNJk26WKRNU5M4UvXp6AP8MgJaSFtuBqdY8sfZTaXZHa39S1bkz9xGbg6X"
});

async function run() {
  try {
    const index = pc.index("n8n1536");
    await index.namespace("").deleteAll();
    console.log("Deleted successfully");
  } catch (err) {
    console.error("Error:", err);
  }
}

run();
